docs/index.md

```diff
@@ -23,7 +23,10 @@ it. The way to do this is explained <a href="https://github.com/pinellolab/CRISP
 <p>
 <b>Step 4: Setting up a batch file for CRISPResso2</b><br>Batch file for a CRISPResso2 run is a .txt file with various 
 columns indicating different parameters for each sample in the run. This batch file should also be placed in the folder 
-created for the CRISPResso2 run. The different columns in a batch file are as described below: 
+created for the CRISPResso2 run. Please refer to the **"Rev_Com_Documentation.html"** to input the appropriate reference and 
```

Comment:
can we make the "Rev_Com_Documentation.html" a link?

Reply:
Yes, I will make it a link once it is available.